packages/cli/src/lib/config.ts

```diff
@@ -10,7 +10,7 @@ const defaultConfig = {
 };
 
 type Config = typeof defaultConfig;
-const location = resolve(homedir(), ".scramjetrc");
+const location = resolve(homedir(), ".scramjet_sth_cli_rc");
```

Comment:
Could we use `.sth-cli.json` similar to `/tmp/sth-id.json` if this is json file?

Reply:
fixed